lib/view/page/idea_details_page.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:idea_tracker/model/idea.dart';
+
+class DetailsScreen extends StatelessWidget {
```

Comment:
Conflicting naming with Screen & Page. The project is using page, so rename to IdeaDetailsPage